### ISSUE TITLE
always returns 'undefined'

### ISSUE THREAD

**Issue by guardian2011** [guilhermefarias/cordova-exif#8]
I have copied and pasted your code for the 'full example' but the console always returns 'undefined'
I have checked the file plugin is installed: Plugin "org.apache.cordova.file" already installed on android.

**Comment by eurokdj** [comment]
I confirm. Just tried with your example, under Cordova 3.4.0 and these core plugin versions : org.apache.cordova.camera 0.3.6 "Camera"
org.apache.cordova.file 1.3.3 "File"

On both Android (real device Samsung Galaxy SII under Android 4.1, emulator Nexus 4 under Android 4.4.4, iPad under iOS 8.3)

exifObjet always returns "undefined"

What are we missing ?

**Comment by michaelfranz89** [comment]
I confirm as well.

**Comment by michaelfranz89** [comment]
I can confirm this problem with the following setup:
Cordova version: 5.1.1
org.apache.cordova.file 1.3.3 "File"
cordova-plugin-camera 1.0.0-dev "Camera"

exifObjet always returns "undefined", any fixes ?

**Comment by ykarthikreddy18** [comment]
I'm also facing same problem.. Did anybody got solution?

**Comment by michaelfranz89** [comment]
Unfortunately not :(

**Comment by ykarthikreddy18** [comment]
Did u got any other plugin.. I'm in badly need of this feature

**Comment by michaelfranz89** [comment]
Sorry, I really can't remember how I did it. It was at my previous employer.

**Comment by ykarthikreddy18** [comment]
ooo.. Thankyou for your response michael..

**Comment by guardian2011** [comment]
I switched to this library instead: https://github.com/exif-js/exif-js

**Comment by ykarthikreddy18** [comment]
Thankyou guardian.. I tried  it. It is working

**Comment by bardu** [comment]
@guardian2011 @ykarthikreddy18 How did you guys manage to get `exif-js`? The only tags I get from an image taken with an iPhone  by calling EXIF.getAllTags(img) are:
> {Orientation: 6, ExifIFDPointer: 38, ColorSpace: 1, PixelXDimension: 3264, PixelYDimension: 2448}

**Comment by ykarthikreddy18** [comment]
Hi @bardu It is working only in android.. On iPhone i'm also getting the same values.. If you got the solution for iphone, please post it here. It will be really helpful. Thankyou.

**Comment by bardu** [comment]
I have tried 
https://github.com/mattiasw/ExifReader
in conjunction with the  Cordova FileReader API but ended up getting the same few tags. I should have done some research prior my implementation which would have saved me some time. It looks like that iPhone strips off Exif data anyway, so there is nothing we can do as it stands of today.

**Comment by ykarthikreddy18** [comment]
Yeah I searched yesterday.. For security reasons they will remove those data.

**Comment by vexsoluciones** [comment]
i've found this 

https://github.com/remoorejr/cordova-plugin-camera-with-exif

they are using exif in ios with success, i have not used yet wil try it
